pages/options.coffee

```diff
@@ -127,6 +129,19 @@ class ExclusionRulesOption extends Option
   getPassKeys: (element) -> element.querySelector(".passKeys")
   getRemoveButton: (element) -> element.querySelector(".exclusionRemoveButton")
 
+  sortRules: ->
+    rules = @readValueFromElement()
+    for rule in rules
+      key = rule.pattern
+      key = RegExp.$1 if 0 == key.search /^[a-z?*]+:\/\/(.*)/
+      key = RegExp.$1 if 0 == key.search /^www\.(.*)/
+      rule.key = key
+
+    rules.sort (a,b) -> if a.key < b.key then -1 else if b.key < a.key then 1 else 0
+    @element.innerHTML = ""
+    @appendRule rule for rule in rules
+    @onUpdated()
```

Comment:
Maybe we should change the button's text to indicat we have done this work.

My exclusion panel has only 2 rules and both urls are too long for the pattern inputs. So I could not see any change after I clicked it.